### ISSUE TITLE
Method references `Async::await` are not taken away

### ISSUE THREAD

**Issue by azurvii** [electronicarts/ea-async#45]
The instrumentation tool takes away all `Async.await` occurrences, but does not take away `Async::await`. I think this is a bug?

**Comment by JoeHegarty** [comment]
Yes, this is likely a bug. A PR would be welcome, unfortunately I am unlikely to get time to fix in the near future.
